packages/components/MenuButton/src/MenuButton.tsx

```diff
@@ -103,10 +117,19 @@ export const MenuButton = compose<MenuButtonType>({
     }
     const context = renderData.state!.context;
     const menuItems = renderData.slotProps!.contextualMenuItems?.menuItems || [];
+    const primary = renderData.slotProps!.root?.primary;
 
     return (
       <Slots.root>
-        <Slots.button />
+        {
+          primary ?
+          <Slots.primaryButton>
+            {getChevronIcon('#fff')}
+          </Slots.primaryButton> :
+          <Slots.button>
+            {getChevronIcon()}
```

Comment:
Could we not pass the SVG into the Button slots' endIcon slot?

Reply:
Yes, I fixed it